lib/getUniqueMatch.js

```diff
@@ -2,19 +2,25 @@
 var Cesium = require('cesium');
 var defined = Cesium.defined;
 
+module.exports = getUniqueMatch;
+
 /** Generic regex search over array of String. (Requires global regex)
```

Comment:
I'm going to merge this so I can finish reviewing #14, you can just make this update in that branch.